scripts/loki_transform.py

```diff
@@ -29,9 +28,6 @@
 )
 
 # pylint: disable=wrong-import-order
-from transformations.argument_shape import (
-    ArgumentArrayShapeAnalysis, ExplicitArgumentArrayShapeTransformation
```

Comment:
These should not have been removed. This is an error in #142 and is fixed in #149. Good catch, and thanks for "flagging" this though. :wink:

Reply:
Apologies for blindly trusting the linter instead of using my brain...